### ISSUE TITLE
Synthetic Medichines Don't Work at All

### ISSUE THREAD

**Issue by KasparoVy** [VOREStation/VOREStation#7588]
#### Brief description of the bug
A synth with the Synth-Only Medichines NIFSoft doesn't actually get repaired at all. You don't get any messages, repairs or anything, regardless of if the damage to a limb is under 5 as described in the code [here](https://github.com/VOREStation/VOREStation/blob/5655573a372576a8ffcbed8c1140790d250f90ce/code/modules/nifsoft/software/05_health.dm#L114)

#### What you expected to happen
I expected my medichines to repair the minor dents and scrapes I had.

#### What actually happened
I stubbed my synthetic toe and my NIFSoft didn't do a thing about it.

#### Steps to reproduce
- (Step 1) Install synth medichines from NIFSoft vendor.
- (Step 2) Take <5 damage to any part of your body.
- (Step 3) Observe bug - medichines do absolutely nothing

#### Code Revision
- e89555cb291cd727bf4ca3d06bd065032403287d 
 
#### Anything else you may wish to add:
- This worked properly last night 20200501 before 12 midnight EST

_Damage readout_
![synth medichines not working at all](https://user-images.githubusercontent.com/12377767/80856040-fcc2c680-8c35-11ea-9292-bbedac809419.PNG)

I also scratched myself with no armour while I had full health & my NIF didn't react at all

I suspect there may be issues with all NIFSofts

**Comment by Arokha** [comment]
![dreamseeker_2020-05-04_20-48-01](https://user-images.githubusercontent.com/15028025/81026590-9cf03980-8e48-11ea-87c5-fd64dc5dbf29.png)
![2020-05-04_20-48-18](https://user-images.githubusercontent.com/15028025/81026593-9f529380-8e48-11ea-95a3-eff2b195493e.gif)

Perhaps you didn't have the nutrition to run them?

**Comment by Arokha** [comment]
Hit me up on discord during a round if it happens again so I can VV you!

**Comment by KasparoVy** [comment]
I was >500 nutrition because of the charger bug, but hopefull it was a one-off-- thanks

**Comment by KasparoVy** [comment]
@Arokha I fucked up and forgot to DM you on discord :(. Happened again today during the shift that started around 7 EST. At first I couldn't use any cyborg chargers, so I was just steadily draining battery (was actually a diff issue in another PR) but got around it by using an inducer -- I also got a NIF to help since there's the APC charging NIFSoft, but during calibration when you fall over and such, I couldn't get up at all and had to have people shake me a few times

I stepped back through a few versions of the Beta BYOND client but that didn't help either, so eventually I threw in the towel and left the shift lol. I'm thankful for a bunch of helpful folks that tried to sort me out (and in the end dragged me to the exit teleporter at my request lol), and it made for at least a bit of RP for Science.

Not sure what's up. Maybe I'm just a bad robot.
